independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json

```diff
@@ -0,0 +1,9216 @@
+                  },
+                  "updated": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "updated"
+                  },
+                  "cases": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "cases"
+                  },
+                  "todayCases": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "todayCases"
+                  },
+                  "deaths": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "deaths"
+                  },
+                  "todayDeaths": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "todayDeaths"
+                  },
+                  "recovered": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "recovered"
+                  },
+                  "active": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "active"
+                  },
+                  "casesPerOneMillion": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "casesPerOneMillion"
+                  },
+                  "deathsPerOneMillion": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "deathsPerOneMillion"
+                  },
+                  "tests": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "tests"
+                  },
+                  "testsPerOneMillion": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "testsPerOneMillion"
+                  },
+                  "population": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "population"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get COVID-19 totals for all US States",
+        "description": "Get COVID-19 totals for all US States",
+        "operationId": "GetCOVID-19totalsforallUSStates",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "sort",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "",
+            "x-ms-visibility": "important",
+            "enum": [
+              "cases",
+              "todayCases",
+              "deaths",
+              "active"
+            ],
+            "x-ms-summary": ""
```

Comment:
@kelcho-spense Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
I have rectified the error.